Description: Fix tests, take 2945
 Every time a new column is added to dysk, a Debian Developer dies
 (fixing tests that are supposed to check columns).
Author: NoisyCoil <noisycoil@debian.org>
Forwarded: no
Last-Update: 2025-10-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/cols.rs
+++ b/src/cols.rs
@@ -221,6 +221,8 @@
                 MountPoint,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
                 Label,
             ],
         );
@@ -241,6 +243,8 @@
                 InodesCount,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
                 Use,
             ],
         );
@@ -260,6 +264,8 @@
                 InodesCount,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
                 Filesystem,
                 Type,
                 Disk,
@@ -294,6 +300,8 @@
                 MountPoint,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
             ],
         );
         check(
@@ -318,6 +326,8 @@
                 MountPoint,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
             ],
         );
     }
@@ -366,6 +376,8 @@
                 InodesCount,
                 Uuid,
                 PartUuid,
+                MountOptions,
+                CompressLevel,
             ],
         );
         check(
